### ISSUE TITLE
How to get error object?

### ISSUE THREAD

**Issue by gijo-varghese** [Yoctol/messaging-apis#356]
When something goes wrong with FB they throw an error with the status code, type etc. How can I access that error object? Right now when I catch it, it's giving me an error with string. Not getting the error object properly

**Comment by tw0517tw** [comment]
Hi @gijo-varghese ,

What did you get when you log the error you caught?

We use the [`axios-error`](https://github.com/Yoctol/messaging-apis/tree/master/packages/axios-error) package to prettify the message output of the error instances so when you `console.log(error)` it can be displayed better. You can still access the `config`, `request` and `response` properties of the error instance.

**Comment by gijo-varghese** [comment]
When something goes wrong, FB returns an error object with the error code, type etc (https://developers.facebook.com/docs/facebook-login/access-tokens/debugging-and-error-handling#errors). Since you're prettifying it, it's not easy to get that error object. It would be great if you return the axios object itself, so we check the status code, parse the error object and so much more.

**Comment by tw0517tw** [comment]
I admit that we could have better documentation on it. The error code from Facebook is in the response body so when you do `error.response.data` you should see them. 

If you check the source code you can see that we try to preserve almost everything from axios and only override the `inspect()` method.

Returning the raw error object sounds good, but as we provide custom construction we might need to think a little more.

**Comment by gijo-varghese** [comment]
Cool. You should document a little more about the error object.

I've tried consoling the error object. But I couldn't find it. Let me try again.

**Comment by tw0517tw** [comment]
I'll keep this open and tag it as doc.
